### ISSUE TITLE
[FEATURE REQUEST] AirPods touch control - Open other App

### ISSUE THREAD

**Issue by Jonyder** [quentinlesceller/macmediakeyforwarder#13]
Hi,
I'm using AirPods Pro. When I use the touch control on those, single press, it opens the Music app when no other audio source is in use. Would be great if this action can also trigger another app (Spotify) to open instead of the Music app.

**Comment by HitoriSensei** [comment]
This is true for any bluetooth headphones. I'm using Sony WH1000XM3, Sony WF1000XM4, some Samsung Buds and cheap Jabra ones.
For each, when pressing "play" it always opens Music app when no music is playing or paused, even if Spotify is already open.
Keyboard buttons work fine tho.

**Comment by quentinlesceller** [comment]
Hi @Jonyder, thanks for the suggestion!

The single press AirPods gesture that opens a music app is handled by macOS itself, not by the media key events this app forwards, so MacMediaKeyForwarder cannot change which app that gesture launches. That part sits outside what this tool can control.

If the goal is to have the media keys (play/pause, next, previous) control Spotify instead of Apple Music, that is already supported: open the menu and choose "Prioritize Spotify". For a broader "support other apps" option, follow #14 where it is being tracked.

Closing this as out of scope, but thanks for taking the time to file it.
